frontend/public/components/RBAC/role.jsx

```diff
@@ -238,6 +245,26 @@ export const BindingsForRolePage = (props) => {
   );
 };
 
+const getBreadcrumbs = (modelRef, kindObj, match) => {
+  const lastNamespace = getLastNamespace();
+  const model = modelFor(modelRef);
+  return [
+    {
+      name: i18next.t(model.labelPluralKey) || model.labelPlural,
+      path: resourceListPathFromModel(
+        model,
+        !lastNamespace || lastNamespace === ALL_NAMESPACES_KEY ? null : lastNamespace,
```

Comment:
Why is the `!lastNamespace` check needed? Won't this pass a `boolean` sometimes to `resourceListPathFromModel` which expects a string when `lastNamespace` is not set?

Reply:
lastNamespace is null before user specifically selects a project.  Basically test is saying "if lastNamespace is null OR not null and equals 'all namespaces', then pass in a `null` namespace string  which `resourceListPathFromModel` will then default to 'all namespaces' when no namespace is passed in.  
If lastNamespace is not null but doesn't equal 'all namespaces', then pass it in to be used by `resourceListPathFromModel`